M2/Macaulay2/packages/Colon/saturate-doc.m2

```diff
@@ -0,0 +1,129 @@
+undocumented {
+    (saturate, Ideal,  Number),
+    (saturate, Vector, Number),
+    (saturate, Module, Number),
+    (saturate, Ideal,  List),
+    (saturate, Module, List),
+    (saturate, Vector, List),
+    (saturate, Vector),
+    (saturate, Vector, Ideal),
+    (saturate, Vector, RingElement)
+    }
```

Comment:
Some of these seem worth documenting.

Reply:
Why? It's just a shorthand for saturating the submodule generated by a single element.

Reply:
It's harmless, so I added them to the documentation.

Reply:
> Why? It's just a shorthand for saturating the submodule generated by a single element.

So the user can learn the shorthand by reading the documentation.

Reply:
This is fixed now. Not sure why github isn't labeling the diff as outdated.